client/src/pages/guide/english/working-in-tech/code-reviews/index.md

```diff
@@ -12,3 +12,9 @@ grow as a developer.
 Is is important to be humble throughout the code review process. Accept your mistakes, 
 and be professional when pointing out improvements or errors in the code of your fellow
 team members.
+
+Code Review have been proven to improve quality, modularization and reduce error-proneness in a software project.
```

Comment:
Please rewrite the first sentence, as it's awkward and has some grammar mistakes.